azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -50,6 +52,13 @@ public void run() throws Exception {
       handleError("Bad property definition! " + e.getMessage(), e);
     }
 
+    Pair<Long, Long> memPair = getProcMemoryRequirement();
```

Comment:
Mem check logic can be disabled by putting "executor.memCheck.enabled=false" in azkaban.properties.